CONTRIBUTING.md

```diff
@@ -233,7 +234,7 @@ A problem must have a unique slug. This slug is used as
 ### In exercism/<TRACK_ID>
 
 * Do the same as when [porting an
-  exercise](#porting-an-exercise-to-another-language-track).
+  exercise](https://github.com/exercism/legacy-docs/blob/main/you-can-help/implement-an-exercise-from-specification.md).
```

Comment:
This link also points to a document with outdated information. It's probably a little bit better than nothing, but we should really be adding this information to our docs website :)